### PR TITLE
Update carousels to use Slick Slider

### PR DESCRIPTION
**Pull request by justinvandenberg**
Ticket: 
#202

Changes:
* Switched all carousels to use Slick Slider
* Removed old carousel JS/CSS

### CODE REVIEW

File: web/themes/npm/assets/styles/elements/slick.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    &.slick-active  > div > div > div {
+      @apply shadow-sm;
+    }
+
+    & > div > div {
+      @apply block !important; /* Slick inline override */
+    }
+  }
+}
```
Comment (mzlock) [inline web/themes/npm/assets/styles/elements/slick.css]: Oof, some of this is some crazy specific and brittle stuff... but you do need to do what you can with a third-party tool like this.

File: web/themes/npm/assets/styles/elements/slick.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+@import "slick-carousel/slick/slick.scss";
+
+.slick-slide[aria-hidden] {
+  @apply block;
+}
+
+[data-label="programs"],
+[data-label="services"],
+[data-label="testimonials"],
+[data-label="testimonials-front-page"] {
```
Comment (mzlock) [inline web/themes/npm/assets/styles/elements/slick.css]: I'm a little iffy on using the data label as a selector.... On the one hand, these shouldn't change, so it should be fine to use them as selectors. On the other, I would never expect a `data-label` to be used for styling. I might change a `data-label` while refactoring and it'd be a surprise that it had a styling effect.

Maybe use classes for these instead? I understand if you don't make this change though - you're not really breaking any standard. This just feels a little unexpected to me.

**Comment by olets** [comment]
Just seeing this. Looks good, Slick is solid and battle tested. For future reference if you wanted to dodge JQuery I've been happy with https://pawelgrzybek.github.io/siema/

**Comment by justinvandenberg** [comment]
@olets I've have to check that out. I knew Slick had all the features I needed, but I forgot it used jQuery.  I also found this one https://swiperjs.com/

**Comment by olets** [comment]
Neat, hadn't seen SwiperJS. Recently saw https://glidejs.com/ in the wild on some app's page, seems nice. What I love about Siema is the dev has stated he thinks carousels are terrible 😆 🥇

**Pull request merged**